.storybook/utils.ts

```diff
@@ -45,4 +45,15 @@ const getLoginUrl = () => {
     }
 };
 
-export { getGridUrl, getImageSigningUrl, getLoginUrl };
+const getPreviewUrl = () => {
+    switch (getEnvironment()) {
+        case 'LOCAL':
```

Comment:
Small question: why do we call it `LOCAL` here, not `DEV`? Do we use the `getEnvironment()` function more widely (in which case: don't fix) or just here?

Reply:
I _think_ it's entirely internal to this file (`getEnvironment` isn't exported). Definitely happy to rename, but I think a separate PR makes sense.